pkg/models/object.go

```diff
@@ -0,0 +1,119 @@
+package models
+
+//-----------------------------------------------------------------------------------------------------
+// NOTE: the object store is in heavy development, and the locations will likely continue to move
+//-----------------------------------------------------------------------------------------------------
+
+import "context"
+
+const (
+	StandardKindDashboard = "dashboard"
+	StandardKindPlaylist  = "playlist"
+	StandardKindFolder    = "folder"
+
+	// StandardKindDataSource: not a real kind yet, but used to define references from dashboards
+	// Types: influx, prometheus, testdata, ...
```

Comment:
i finally did get around to [writing down my concerns](https://github.com/grafana/grafana/pull/56492#discussion_r990067526) about the simple kind/type relation